tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -264,6 +270,17 @@ install_kernel_tdx_experimental() {
 		"-x tdx -u ${kernel_url}"
 }
 
+#Install sev kernel asset
+install_kernel_sev() {
+	info "build sev kernel"
+	local kernel_url="$(yq r $versions_yaml assets.kernel.sev.url)"
```

Comment:
Please, do not use  `yq r`, but use `get_from_kata_deps` instead, as already done through out this file.

Reply:
@UnmeshDeodhar, please, do not use yq r, but use get_from_kata_deps instead, as already done through out this file.

This fix should be part of 0a6f0ab3cbdce05d686ebb5045a57511e8c77ce2, not in a different one.